data/mods/Xedra_Evolved/monsters/elementals.json

```diff
@@ -761,6 +761,129 @@
+    "weight": "61500 g",
+    "hp": 80,
+    "speed": 70,
+    "material": [ "fae_flesh" ],
+    "symbol": "@",
+    "color": "light_red",
+    "aggression": 30,
+    "morale": 70,
+    "aggro_character": false,
+    "melee_skill": 5,
+    "melee_dice": 2,
+    "melee_dice_sides": 5,
+    "melee_damage": [ { "damage_type": "heat", "amount": 4 } ],
+    "special_attacks": [ { "id": "salamagman_handaxe" } ],
+    "dodge": 2,
+    "harvest": "fae_changeling",
+    "dissect": "dissect_salamander_large",
+    "weakpoint_sets": [ "wps_alien_biology" ],
+    "families": [ "prof_alien_biology" ],
+    "anger_triggers": [ "FRIEND_ATTACKED", "FRIEND_DIED", "HURT" ],
+    "path_settings": { "avoid_traps": true, "avoid_sharp": true },
+    "death_drops": { "subtype": "collection", "groups": [ [ "mon_salamagman_death_drops", 40 ] ] },
+    "flags": [ "SEES", "HEARS", "WARM", "BASHES", "PATH_AVOID_DANGER", "PATH_AVOID_FALL", "FIREPROOF" ],
+    "armor": {
+      "bash": 16,
+      "cut": 12,
+      "bullet": 26,
+      "xe_cold_iron_cut_damage": 0,
+      "xe_cold_iron_bash_damage": 0,
+      "xe_cold_iron_stab_damage": 0
+    }
+  },
+  {
+    "id": "mon_salamagman_large",
+    "type": "MONSTER",
+    "name": { "str": "salamagman champion" },
+    "description": "A sluglike lower body made of barely cooled magma transforms into a glowing red, squat, barrel-like torso and cooling magma forms a beard and hair around its face.  A black mirror battleaxe reflects your image darkly back at you.",
+    "default_faction": "salamander",
+    "bodytype": "gastropod_mutant",
+    "species": [ "SALAMANDER" ],
+    "volume": "102500 ml",
+    "weight": "121500 g",
+    "hp": 120,
+    "speed": 70,
+    "material": [ "fae_flesh" ],
+    "symbol": "@",
+    "color": "light_red",
+    "aggression": 30,
+    "morale": 70,
+    "aggro_character": false,
+    "melee_skill": 5,
+    "melee_dice": 2,
+    "melee_dice_sides": 5,
+    "melee_damage": [ { "damage_type": "heat", "amount": 4 } ],
+    "special_attacks": [ { "id": "salamagman_battleaxe" } ],
+    "dodge": 2,
+    "harvest": "fae_changeling",
+    "dissect": "dissect_salamander_large",
+    "weakpoint_sets": [ "wps_alien_biology" ],
+    "families": [ "prof_alien_biology" ],
+    "anger_triggers": [ "FRIEND_ATTACKED", "FRIEND_DIED", "HURT" ],
+    "path_settings": { "avoid_traps": true, "avoid_sharp": true },
+    "death_drops": { "subtype": "collection", "groups": [ [ "mon_salamagman_champion_death_drops", 40 ] ] },
+    "flags": [ "SEES", "HEARS", "WARM", "BASHES", "PATH_AVOID_DANGER", "PATH_AVOID_FALL", "FIREPROOF" ],
+    "armor": {
+      "bash": 16,
+      "cut": 12,
+      "bullet": 26,
+      "xe_cold_iron_cut_damage": 0,
+      "xe_cold_iron_bash_damage": 0,
+      "xe_cold_iron_stab_damage": 0
+    }
+  },
+  {
+    "id": "mon_salamagman_warlord_butterfly",
+    "type": "MONSTER",
+    "name": { "str": "salamagman warlord, obsidian butterfly", "str_pl": "salamagman warlords, obsidian butterflies" },
```

Comment:
Too long for a name, sidebar and compass won't display this nicely. Shorten it to 3 words or less?